in-toto-spec.md

```diff
@@ -1393,15 +1500,14 @@ verification and install Alice's `foo.py` script.
 
 When Carl is verifying, his  installer will perform the following checks:
 
-
 1. The root.layout file exists and is signed with a trusted key (in this case,
    Alice's).
 1. The code that Alice wrote was tested by two people, in this case, Caroline
    and Alfred.
-1. Every step in the layout has a corresponding [name].link metadata file
+1. Every step in the layout has a corresponding [name].[keyid-prefix].link metadata file
```

Comment:
Should put [name].[keyid-prefix].link between quotes to obtain the grayed text effect.
(Same for later in this paragraph)

Reply:
Right. Will do.